modules/api/pkg/handler/v2/routes_v2.go

```diff
@@ -1439,6 +1439,10 @@ func (r Routing) RegisterV2(mux *mux.Router, oidcKubeConfEndpoint bool, oidcCfg
 		Path("/quotas/{quota_name}").
 		Handler(r.getResourceQuota())
 
+	mux.Methods(http.MethodGet).
+		Path("/totalquota").
```

Comment:
It seems that GET `quotas` endpoint is still available (you need to double check me on that). If so, how about using it for returning the final sum for all quotas?
```suggestion
		Path("/quotas").
```